src/lib/util/Adder.ts

```diff
@@ -1,3 +1,13 @@
+export class AdderError extends Error {
+
+	public amount: number;
+	public constructor(message: string, amount: number) {
```

Comment:
```suggestion
	public constructor(message: string, public amount: number) {
```

I like to keep classes that extend error for extra properties nice and tidy (yes this is valid, I learned that because Angular does this by default)

Reply:
I'm confident ESLint's rules forbid that (I tried a while ago). Either way I documented the property and the constructor, which wouldn't be possible without this.